src/templates/results/components/ObservedTable.js

```diff
@@ -0,0 +1,82 @@
+import { Row, Col, Table } from 'react-bootstrap';
+
+
+const ObservedTable = (props) => {
+    const observedTaxa = props.observedTaxa;
+    const chosenTaxon = props.chosenTaxon;
+
+    return (
+        <Row className="h-100">
+            <Col className="h-100">
+                <Row className="results_titleBar">
+                    <Col className="results_titleBarSub col-md-auto fw-bold h-100">
+                        Observed interactions
+                    </Col>
+                </Row>
+                <Row className="results_tableSection">
+                    <Col className="h-100">
+                        <Table striped className="results_table h-100">
```

Comment:
Probably not necessary here but there are a couple of good libs out there for tables. Worked with agGrid previously but another option could be the ant table checklist bank uses. Might be a bit overkill for here but know they are there.

Reply:
Good to know! Until now been using the Bootstrap Table, which is pretty simple. Could be more interesting for UCAS as it uses a library for table sorting which is outdated (and causes some issues with legacy peer deps)